### PR TITLE
feat(i18n): update translations

### PR DESCRIPTION
**Pull request by tiborux**
EX-6461

### CODE REVIEW

File: src/i18n/messages.types.ts [hunk 1]
```diff
@@ -13,42 +13,33 @@ export interface Messages {
...
   priceFilter: {
     lessThan: string;
     fromTo: string;
     from: string;
   };
-  filtersSearch: {
-    placeholder: string;
-  };
   toggleAside: {
-    hideAside: string;
```
Comment (CachedaCodes) [inline src/i18n/messages.types.ts:51]: This one is still being used in the mobile layout.

File: src/i18n/messages/en.messages.json [hunk 2]
```diff
@@ -66,7 +57,7 @@
       "message": "No results found for <span class=\"x-text x-text--accent x-text--bold\">'{query}'</span>. We show you results for"
     },
     "noResults": {
-      "message": "No results found for <span class=\"x-text x-text--accent x-text--bold\">'{query}'</span>. Try with another query."
+      "message": "Sorry but there are no results for <span class=\"x-text x-text--accent x-text--bold\">'{query}'</span>. You may be interested in these products"
```
Comment (CachedaCodes) [inline src/i18n/messages/en.messages.json:60]: This is not the message in the designs for the `no-results` (no dot after the first phrase and a line break instead). Should be changed here or there's a task for it? (A little styling is required).

image

File: src/i18n/messages/en.messages.json [hunk 3]
```diff
@@ -12,42 +12,33 @@
...
-      "usage": "USAGE",
-      "season": "SEASON"
+      "price": "PRICE"
     },
     "filters": {
-      "all": "All"
+      "all": "Show more"
+    },
+    "partialResults": {
+      "message": "VIEW ALL"
```
Comment (CachedaCodes) [inline src/i18n/messages/en.messages.json:34]: This is not the message for the partials in the designs. If we should change it in this task it should be:

```suggestion
      "message": "View all results ({totalResults})"
```

image

**Review by CachedaCodes — CHANGES_REQUESTED** [review]

File: src/i18n/messages/en.messages.json [hunk 4]
```diff
@@ -12,42 +12,33 @@
...
       "gender": "GENDER",
       "categoryPaths": "CATEGORIES",
-      "price": "PRICE",
-      "usage": "USAGE",
-      "season": "SEASON"
+      "price": "PRICE"
     },
     "filters": {
-      "all": "All"
+      "all": "Show more"
```
Comment (mnavarroespinosa) [inline src/i18n/messages/en.messages.json:31]: Shouldn't this still be 'All'?

```suggestion
      "all": "All"
```

**Review by CachedaCodes — APPROVED** [review]

**Review by mnavarroespinosa — APPROVED** [review]

**Pull request merged**